setup.cfg

```
@@ -16,4 +16,18 @@ profile = black
 combine_as_imports = True
 
 [tool:pytest]
-addopts = --cov-report= --cov=starlette --cov=tests -rxXs
+addopts =
+  --cov-report=
```

any reason for not using

```suggestion
  --cov-report=term-missing
```

or
```suggestion
  --cov-report=term-missing:skip-covered
```

---

getting the missing line numbers is handy for running locally https://pytest-cov.readthedocs.io/en/latest/reporting.html